### ISSUE TITLE
Signal 11 Error

### ISSUE THREAD

**Issue by realsurya** [jbuchermn/newm#109]
Hey devs,

I was just trying out newwn and it seems very cool and interesting. However, I get intermittent crashing due "Error: Signal 11" similar to #106. I'm usually able to use the compositor for a couple minutes before it crashes due to error 11.

Not sure if you actually got the logfile from #106, so I've attached a logfile here for reference. Hope this helps!

[newm_log.txt](https://github.com/jbuchermn/newm/files/8648603/newm_log.txt)

**Comment by jbuchermn** [comment]
Thanks for the log!

Can you see any patterns? I.e. does it crash when you close a window, or something similar?

**Comment by realsurya** [comment]
Yep there seems to be a very peculiar pattern. I did a few tests today:

1) Starting newm and doing nothing.
2) Starting newm and opening a single terminal (Alacritty) and using HTOP.
3) Starting newm and opening terminal & Firefox.

On all three trials, newm took 1 min 16 seconds to crash to error 11, which is interesting. I've attached all three logfiles just in case you're interested, since it's bizarre the timing seems so consistent. 

My machine specs are:
- Microsoft Surface Book 2 (res 3240×2160)
- Intel Core i7-8650U (4 cores, 8 threads)
- 16 GB ram
- GTX 1060M GPU (No Nvidia drivers, though)
- Distro: Arch Linux, got newm through yay.

[newm_log_Firefox.txt](https://github.com/jbuchermn/newm/files/8653641/newm_log_Firefox.txt)
[newm_log_OnlyUsingHTOP.txt](https://github.com/jbuchermn/newm/files/8653642/newm_log_OnlyUsingHTOP.txt)
[newm_log_WhenDoingNothing.txt](https://github.com/jbuchermn/newm/files/8653643/newm_log_WhenDoingNothing.txt)

**Comment by jbuchermn** [comment]
Thanks a lot, that helps.

I haven't yet been able to reproduce, but the logs and you're patterns suggest it might have to do something with the panels (which are started automatically in the background when they finish). Somehow starting the launcher panel fails and the interaction between the panel starting and the compositor triggers the bug (segfault).

To verify this thought - can you try disabling the panels and post a log?

```
panels = {
   `launcher`: { `cmd`: None },
   `lock`: { `cmd`: None }
}
```

**Comment by jbuchermn** [comment]
Also, can you check for `~/.cache/newm_panel_log` and post it?

**Comment by jbuchermn** [comment]
Im finally able to reproduce! :) happens for me only when `encourage_csd` is set

**Comment by jbuchermn** [comment]
This should be fixed by https://github.com/jbuchermn/pywm/commit/7980012191e953440eca87e168c81d8185344c39
